src/main/java/frc/robot/subsystems/AlgaeSubsystem.java

```diff
@@ -64,12 +73,25 @@ public AlgaeSubsystem() {
                                    .forwardSoftLimit(0) 
                                    .forwardSoftLimitEnabled(true);
 
+        // Get pid values from network tables
+        NetworkTable AlgaeTuningTable = NetworkTableInstance.getDefault().getTable(m_tuningTable);
+        m_AlgaeP = 0.2;
```

Comment:
should eventually clean up these hard coded constants, but let's do that after NB